### PR TITLE
Some small additions.

### PR DESCRIPTION
**Pull request by styroxfjant**
Test sentence was missing for fan_HassTurnOff, so I had to add that also.

### CODE REVIEW

File: sentences/sv/_common.yaml [hunk 1]
```diff
@@ -59,10 +59,10 @@ expansion_rules:
   är: "(är det | är)"
   brightness: "{brightness} [percent]"
   slå_på: "(tänd | sätt (på | igång | fart på) | slå (på | till) | starta)"
-  slå_av: "(slå (av | från | ifrån) | släck | stäng av | stoppa)"
+  slå_av: "(slå (av | från | ifrån) | släck | stäng (av) | stoppa)"
```
Comment (larsdunemark) [inline sentences/sv/_common.yaml:62]: Should be "stäng [av]" to make "av" optional.

File: sentences/sv/_common.yaml [hunk 2]
```diff
@@ -59,10 +59,10 @@ expansion_rules:
...
   i_på: "(i | på | vid | bredvid)"
   ändra: "(ställ in | sätt | justera | ändra | vrid (upp | ner))"
-  temp: "(temperatur)"
+  temp: "(temperatur[en])"
```
Comment (larsdunemark) [inline sentences/sv/_common.yaml:65]: I don't think this is used, in that case we should add a test case. It's a good idea to separate different intents/changes into separate PR.

**Review by larsdunemark — CHANGES_REQUESTED** [review]

File: sentences/sv/_common.yaml (hunk 1 above)
Comment (styroxfjant) [reply sentences/sv/_common.yaml:62]: You are correct.

File: sentences/sv/_common.yaml (hunk 2 above)
Comment (styroxfjant) [reply sentences/sv/_common.yaml:65]: I'll remove this for now then. If seen needed we must add it + the test cases later.

**Review by larsdunemark — APPROVED** [review]

**Pull request merged**